src/operator/contrib/transformer-inl.h

```diff
@@ -34,6 +34,18 @@
 namespace mxnet {
 namespace op {
 
+struct InterleavedMatMulParam : public dmlc::Parameter<InterleavedMatMulParam> {
+  int heads;
+  bool bwd_ignore_zero_init;
+  DMLC_DECLARE_PARAMETER(InterleavedMatMulParam) {
+    DMLC_DECLARE_FIELD(heads)
+    .describe("Set number of heads");
+    DMLC_DECLARE_FIELD(bwd_ignore_zero_init)
+    .describe("Make backward pass ignore AddTo and not init to 0.")
```

Comment:
I don't think this one sentence explains this flag very well. When are users supposed to turn this on? Would it affect users who use gradient accumulation for self-attention?

Reply:
Sure, it's kind of tricky to explain. Until we don't have the gradient accumulation feature working the user should not use this flag, it's cheating on purpose relying on the fact that the two ops of self attention are using complementary parts of the input tensor (not overlapping and the sum of all is the full tensor). If you want to use gradient accumulation with another Op you should not enable this flag. Any indication on wording would be appreciated.

Reply:
can you add more of what you said to the description? this is good info